<!--
<node pkg="tf" type="static_transform_publisher" name="broadcasterTFmini" args="0 0 0 0 0 0 base_link tfmini_link 100" />
-->

<launch>

  <node pkg="px4_command" type="TFmini" name="TFmini" output="screen">
    <param name="serial_port" type="string" value="/dev/tfmini" />
    <param name="baud_rate" type="int" value="115200" />
    <param name="frame_id" type="string" value="tfmini_link"/>
  </node>
  

</launch>
